docs/changes/io.misc/15373.bugfix.rst

```diff
@@ -0,0 +1 @@
+To avoid segmentation faults when dumping/loading ``yaml`` files, trying to dump a numpy object array to ``yaml`` or loading it will now rise a ``TypeError``
```

Comment:
```suggestion
To avoid segmentation faults when dumping/loading ``yaml`` files, trying to dump a numpy object array to ``yaml`` or loading it will now rise a ``TypeError``.
```